modules/mssql/main.tf

```diff
@@ -122,7 +122,8 @@ resource "google_sql_database_instance" "default" {
     user_labels = var.user_labels
 
     location_preference {
-      zone = var.zone
+      zone           = var.zone
+      secondary_zone = var.secondary_zone
```

Comment:
Looks like this was added in TPG [4.28.0](https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md). Can we also bump the required provider versions for all modules? https://github.com/terraform-google-modules/terraform-google-sql-db/blob/95e48a1473714fc5a5ba46b4dce69b21bb6e00d7/modules/mysql/versions.tf#L28-L31

Reply:
done!